pandas/tests/series/indexing/test_set_value.py

```diff
@@ -19,3 +19,24 @@ def test_series_set_value():
     expected = Series([1.0, np.nan], index=index)
 
     tm.assert_series_equal(s, expected)
+
+
+def test_set_value_dt64(datetime_series):
```

Comment:
you really want to create a module named this? e.g. i would put this in setitem / setloc (not sure if we have that)

Reply:
its for the `Series._set_value` method.  next time you want to write a test for that method you'll know exactly where it goes.

Reply:
ahh ok, yeah that's the biggest thing is to put new tests in the right place.